src/main/java/mryapper/task/Event.java

```diff
@@ -0,0 +1,33 @@
+package mryapper.task;
+
+/**
+ * An event task that occurs within the specified time.
+ */
+public class Event extends Task {
+    private final String eventStart;
```

Comment:
Can just name as start and end (But NO violation here, just thought it sounds more intuitive).